src/material/chips/chip-row.ts

```diff
@@ -104,6 +114,15 @@ export class MatChipRow extends MatChip implements AfterViewInit {
       if (this._isEditing && !this._editStartPending) {
         this._onEditFinish();
       }
+      this._alreadyFocused = false;
+    });
+    this._onFocus.pipe(takeUntil(this.destroyed)).subscribe(() => {
+      if (!this._isEditing && !this.disabled) {
+        // Need a timeout to ensure flag not set while handling the first click.
+        this._ngZone.runOutsideAngular(() => {
+          setTimeout(() => (this._alreadyFocused = true), 100);
```

Comment:
I think we can do this without the timeout by checking whether the chip is focused on the `mousedown`.

Reply:
Great idea! That works well and actually handles the long press better as well. Well, I could have argued that a long press to edit is a feature, but I won't now.

Only thing is then the spec needs to have a mousedown as well.